media-player.js

```diff
@@ -865,6 +908,40 @@ class MediaPlayer extends FocusVisiblePolyfillMixin(InternalLocalizeMixin(RtlMix
 		return a.href;
 	}
 
+	_getChapterMarkersView() {
+		if (!(this._chapters?.length > 0)) return;
```

Comment:
`this.chapters` is initialized to an array - unless there are cases where it can be null?
```suggestion
		if (this._chapters.length === 0) return;
```